.github/dependabot.yml

```diff
@@ -26,3 +26,6 @@ updates:
   - dependency-name: "prettier"
   # newer versions of material ui (MUI) cause our styling to not respect theming or straight up break the UI (DR-2952)
   - dependency-name: "@mui/*"
+  # @mui/styles is incompatible with react 18, we should replace @mui/styles with tss-react (DR-3428)
+  - dependency-name: "@types/react"
+    update-types: [ "version-update:semver-major" ]
```

Comment:
Do you know if just pinning just the react types library going to be sufficient to keep react on v17? I see in the package.json that we also have a reference to react itself.

Reply:
I _think_ that the typescript version of the react dependency is basically the shim to pull in react?

Reply:
Great point, @snf2ye -- we are going to have to pin quite a bit more here.  I've converted the PR back to draft.